tools/hdf_dump.py

```diff
@@ -161,6 +176,8 @@ def hdf_dump_from_dataset(dataset, hdf_dataset, parser_args):
     for data_key in data_keys:
       if data_key == default_data_input_key:
         hdf_data = hdf_dataset['inputs']
+      elif data_key == default_data_target_key:
+        hdf_data = hdf_dataset['targets/data'][valid_hdf_target_key]
       else:
         hdf_data = hdf_dataset['targets/data'][data_key]
```

Comment:
Instead of `data_key` here, you would use `hdf_data_key_map[data_key]`.

Reply:
Done.